src/bmeg/models/__init__.py

```diff
@@ -0,0 +1,14 @@
+import pkgutil
+import inspect
+
+__all__ = []
+
+for loader, name, is_pkg in pkgutil.walk_packages(__path__):
+    module = loader.find_module(name).load_module(name)
+
+    for name, value in inspect.getmembers(module):
+        if name.startswith('__'):
+            continue
+
+        globals()[name] = value
+        __all__.append(name)
```

Comment:
This is duplicating module loads, which makes types incomparable.

e.g.
```
    m = Individual(row["id"], {
        "diagnoses": row["diagnoses"],
    })
    print(m, isinstance(m, Vertex), isinstance(m, V2))
```

which leads to errors like:
```
Traceback (most recent call last):
  File "gdc-cases.py", line 82, in <module>
    emitter.emit(m)
  File "/Users/buchanae/src/github.com/bmeg/bmeg-etl/src/bmeg/models/emitter.py", line 22, in emit
    raise TypeError("emit accepts objects of the Vertex or Edge type")
TypeError: emit accepts objects of the Vertex or Edge type
```

I think we should just manually add entries to the __all__ list, or just import directly from the individual models.

Reply:
Or just keep them all in one file. I'm not against large files when the code in that file is dead simple (true, in this case). That way you wouldn't have to guess which file a certain model is in.